bin/skeleton-chart.sh

```diff
@@ -1,29 +1,24 @@
 #!/usr/bin/env bash
-set -e
```

Comment:
This is bad practise the script will continue even if it fails at some point.

Reply:
yagni...those flags are not applicable. No pipes are used. No env vars.